lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -141,18 +141,15 @@
 
     .action-label {
       @extend %t-copy-sub2;
-      display: inline-block;
+      display: none;
```

Comment:
The opacity was an issue because it was rendering white on white, but the screen reader still read it. I made it ``display: none`` because I don't want anyone to see it, screen readers included. The screen reader will read it when it is given the focus.

@clrux what's your opinion? Should the hiding be a visual-only thing, and just have the screenreader always read it?